pkg/version/cobra.go

```diff
@@ -109,10 +109,10 @@ func CobraCommandWithOptions(options CobraOptions) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().BoolVarP(&short, "short", "s", false, "Displays a short form of the version information")
+	cmd.Flags().BoolVarP(&short, "short", "s", true, "Displays a short form of the version information")
```

Comment:
Instead of changing _pkg/version_, which will affect all CLIs, we should try to only affect _istioctl_ and leave the other CLIs the same.

To do this, change the line in _istioctl/main.go_ `rootCmd.AddCommand(version.CobraCommandWithOptions(version.CobraOptions{GetRemoteVersion: getRemoteInfo}))`.  What you can do is break this up so that before calling AddCommand() you call PersistentFlags() on VersionCmd and change the defaults.  Similar to how we hide logging options in the root command.

Reply:
Thanks a lot Edward, I have submitted the new change according to your comments. Please review.